.github/workflows/main.yml

```diff
@@ -0,0 +1,17 @@
+name: CI / CD
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This is the main branch CI/CD

It runs on pushes to the main branch only

Below you'll see if uses the two other workflows we have and makes sure the tests run BEFORE the deploy starts